src/css/components/_adaptive.scss

```diff
@@ -1,44 +1,70 @@
-// When the player is absurdly tiny, display nothing but:
+// When the player is "medium" and higher, display everything by default.
+//
+// When the player is "small", display only:
 // - Play button
+// - Volume Mute button
+// - Progress bar
+// - Subs-Caps button
+// - Fullscreen button
+//
+// When the player is "small", display only:
+// - Play button
+// - Volume Mute button
+// - Progress bar
+// - Fullscreen button
+//
+// When the player is "tiny", display only:
+// - Play button
+// - Volume Mute button
 // - Fullscreen Button
-.video-js.vjs-layout-tiny:not(.vjs-fullscreen) {
-  .vjs-custom-control-spacer {
-    @include flex(auto);
-    display: block;
-  }
+//
+.video-js:not(.vjs-fullscreen) {
 
-  &.vjs-no-flex .vjs-custom-control-spacer { width: auto; }
+  &.vjs-layout-small,
+  &.vjs-layout-x-small,
+  &.vjs-layout-tiny {
+    .vjs-current-time,
+    .vjs-time-divider,
+    .vjs-duration,
+    .vjs-remaining-time,
+    .vjs-playback-rate,
+    .vjs-chapters-button,
+    .vjs-descriptions-button,
+    .vjs-captions-button,
+    .vjs-subtitles-button,
+    .vjs-audio-button,
+    .vjs-volume-control {
+      display: none;
+    }
 
-  .vjs-current-time, .vjs-time-divider, .vjs-duration, .vjs-remaining-time,
-  .vjs-playback-rate, .vjs-progress-control,
-  .vjs-mute-control, .vjs-volume-control, .vjs-volume-panel,
-  .vjs-chapters-button, .vjs-descriptions-button, .vjs-captions-button,
-  .vjs-subtitles-button, .vjs-subs-caps-button, .vjs-audio-button { display: none; }
-}
+    // Reset the size of the volume panel to the default so we don't see a big
+    // empty space to the right of the mute button.
+    .vjs-volume-panel.vjs-volume-panel-horizontal:hover,
+    .vjs-volume-panel.vjs-volume-panel-horizontal:active,
+    .vjs-volume-panel.vjs-volume-panel-horizontal.vjs-slider-active {
+      width: 4em;
+    }
```

Comment:
This is part of the functional change.

Reply:
We should test to see if `initial` works in firefox, safari, ie11 here instead. This way, we won't specify the width of the control in yet another place. Seems to work for me in Chrome

Reply:
I'll do that.

Reply:
Does not work in IE11. Works everywhere else... :(